docs/samples/python/transformers_recognizer.py

```diff
@@ -0,0 +1,231 @@
+import logging
+from typing import Optional, List, Tuple, Set
+
+from presidio_analyzer import (
+    RecognizerResult,
+    EntityRecognizer,
+    AnalysisExplanation,
+)
+from presidio_analyzer.nlp_engine import NlpArtifacts
+
+try:
+    from transformers import (
+        AutoTokenizer,
+        AutoModelForTokenClassification,
+        pipeline,
+        models,
+    )
+    from transformers.models.bert.modeling_bert import BertForTokenClassification
+except ImportError:
+    print("transformers is not installed")
```

Comment:
NIT: are we sure we want to use print and not logger? the logger can be initialized earlier if needed.

Reply:
@shiranr  Thanks.
Changed to logger.error()